synapse/models/telco.py

```diff
@@ -326,25 +308,21 @@ async def _normPyInt(self, valu, view=None):
 
                 ('mnc', ('tel:mob:mnc', {}), {
                     'ro': True}),
-
```

Comment:
Notes say to add :org:name and :org:place:loc to tel:carrier? If we are adding props, should tel:carrier still be a comp vs a guid (existing 3.0 story to discuss making a guid, this is also green in the notes?)